yargs/play/agent_benchmark.yaml

```diff
@@ -0,0 +1,5 @@
+benchmark_B5W3:
+  args: -t 100 -N 5 -W 3 --players greedy greedy:p_random=0.1,nick=greedy-ish sb3ppo:wandb_alias=latest dexp:wandb_alias=best
```

Comment:
Looks good. Do you think we'll keep the `agent:variable=val,...` syntax now that we have yaml files? Or could those be more like:
```
args:
    agents:
           agent_1_nick:
                - agent_type: greedy
                - p_random: 0.1
                - ....
```
Or something similar? (not in this pr, though)
```

Reply:
For the command line I'm planning to leave it like it is.  For the yaml file, I actually started doing something like the above but it got tricky because sometimes you have lists, dicts, single values, etc, so I aborted thinking that we may not need it, but I'd be happy to implement it, and still support the current format in yaml files, which has the advantage that you can just copy paste it from the command line